docs/book/en/src/extend/build_for_RISC-V64.md

```diff
@@ -0,0 +1,196 @@
+# Build and test WasmEdge on RISC-V 64 arch
+
+Please follow this tutorial to make a RISC-V 64 filesystem with linux kernel, build WasmEdge with riscv-gnu-toolchain and test WasmEdge on the RISC-V64 system.
+
+> Currently, we only support the runtime for the interpreter mode.
+
+## Prepare the Environment
+
+This tutorial is based on Ubuntu 21.10.
+
+### Dependencies
+
+We need to install the following dependencies to support subsequent work.
+
+```bash
+$ sudo apt install autoconf automake autotools-dev curl libmpc-dev libmpfr-dev libgmp-dev \
+                 gawk build-essential bison flex texinfo gperf libtool patchutils bc \
+                 zlib1g-dev libexpat-dev git \
+                 libglib2.0-dev libfdt-dev libpixman-1-dev \
+                 libncurses5-dev libncursesw5-dev
+```
+
+### Download and build riscv-gnu-toolchain
+
+First,we get the riscv-gnu-toolchain source code.It's noted that before downloading the complete submodule, in order to save time, we delete the `qemu` submodule in advance.
+
+```bash
+$ git clone https://github.com/riscv-collab/riscv-gnu-toolchain.git
+$ cd riscv-gnu-toolchain
+$ git rm qemu
+$ git submodule update --init --recursive
+```
+
+Then we configure the installation path of the toolchain and build the toolchain.
+
+```bash
+$ ./configure --prefix=/opt/riscv64 
+$ sudo make linux -j $(nproc)
+```
+
+### Download and build qemu
+
+Get the qemu code and compile qemu. Laterly, we will use `qemu` to make the system image and emulate the RISC-V 64 environment.
+
+```bash
+$ wget https://download.qemu.org/qemu-7.0.0.tar.xz
+$ tar xvJf qemu-7.0.0.tar.xz
+$ cd qemu-7.0.0/
+$ ./configure --target-list=riscv64-softmmu,riscv64-linux-user --prefix=/opt/qemu
+$ make -j $(nproc)
+$ sudo make install
+```
+
+### Download and compile linux kernel
+
+Download the tar.gz archive of linux-v5.14 and rename the extracted folder to `linux`.
+
+```bash
+$ wget https://github.com/torvalds/linux/archive/refs/tags/v5.14.tar.gz
+$ tar xzvf v5.14.tar.gz
+$ mv linux-v5.14 linux
+```
```

Comment:
Looks like these are static versions. It should be able to package it into a docker image, so users don't need to do these stuff by themselves.